.gitignore

```diff
@@ -6,6 +6,7 @@ target/
 .idea/modules.xml
 .idea/jarRepositories.xml
 .idea/compiler.xml
+.idea/uiDesigner.xml
```

Comment:
you can add the whole .idea folder here, there is never a need to push anything from it